javascript/javascript1/week3/homework4.js

```diff
@@ -0,0 +1,63 @@
+// Smart-ease - Back to the basics!
+// NOnoN0nOYes (Note taking app)
+
+// The saveNote function should push an object to the notes array with the keys content and id.
+
+/**
+ *
+ *
+ * @param {string} content
+ * @param {number} id
+ */
+
+const notes = [];
+
+function saveNote(content, id) {
+  notes.push({
+    content: content,
+    id: id,
+  });
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+//unique feature of colored log
+console.log("\x1b[31m%s\x1b[0m", "\n\t SaveNote task \n");
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+
+// Get a note
+
+function getNote(id) {
+  let result = "";
+  for (let i = 0; i < notes.length; i++) {
+    if (notes[i].id === id) {
+      return (result = notes[i]);
```

Comment:
Here you just want to return the note in the array you have iterated over.

Therefore you just want to return:
`return notes[i];`